manticore/platforms/evm.py

```diff
@@ -2165,8 +2246,10 @@ def CALL(self, gas, to, value, data):
     def RETURN(self, data):
         prev_vm = self._pop_vm() #current VM changed!
         if self.depth == 0:
-            self.last_return=data
-            self.last_pc = prev_vm.pc
+            tx = self._transactions[-1]
+            tx.return_data=data
+            #tx.last_pc = prev_vm.pc
```

Comment:
why don't we record the prev_vm.pc in the new version?

Reply:
It's not really part of a transaction. So there is not room for it in tx. 
and... it was not used?. I could not answer: Why do we used to save it ?